polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -39,9 +39,13 @@ namespace polygeist {
 using namespace mlir;
 using namespace mlir::polygeist;
 
-static llvm::cl::list<unsigned> LoopInternalizationTileSizes(
-    DEBUG_TYPE "-tile-sizes", llvm::cl::CommaSeparated,
-    llvm::cl::desc("Tile sizes used in LoopInternalization"));
+static llvm::cl::list<unsigned>
+    TileSizes(DEBUG_TYPE "-tile-sizes", llvm::cl::CommaSeparated,
+              llvm::cl::desc("Tile sizes used in LoopInternalization"));
+
+static llvm::cl::opt<unsigned>
+    LocalMemSize(DEBUG_TYPE "-local-mem-size", llvm::cl::init(32000),
+                 llvm::cl::desc("The size of local memory arena in bytes"));
```

Comment:
Change to: "The minimum size of the shared local memory arena in bytes"

Reply:
Can I change to `The size of the shared local memory arena in bytes"? as it is an option, and user may know the actual size.

Reply:
yes